### PR TITLE
V1v2 migration owner setup

### PR DESCRIPTION
**Pull request by aeolianeth**
## What does this PR do and why?

Part 1 of https://github.com/jbx-protocol/juice-interface/issues/1056

## Screenshots or screen recordings

_If applicable, provide screenshots or screen recordings to demonstrate the changes._

## Acceptance checklist

- [ ] I have evaluated the [Approval Guidelines](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#approval-guidelines) for this PR.
- [ ] I have tested this PR in [all supported browsers](https://github.com/jbx-protocol/juice-interface/blob/main/CONTRIBUTING.md#supported-browsers).
- [ ] I have tested this PR in dark mode and light mode (if applicable).

### CODE REVIEW

File: src/components/shared/Project/ProjectToolsDrawer/V1TokenMigrationSection/V1TokenMigrationModal/SetV1ProjectSection.tsx [hunk 1]
```diff
@@ -0,0 +1,91 @@
+import { Trans } from '@lingui/macro'
```
Comment (aeolianeth) [inline src/components/shared/Project/ProjectToolsDrawer/V1TokenMigrationSection/V1TokenMigrationModal/SetV1ProjectSection.tsx:1]: This component renders this UI:

Screen Shot 2022-07-02 at 7 17 23 PM

File: src/components/shared/Project/ProjectToolsDrawer/V1TokenMigrationSection/V1TokenMigrationModal/AddTerminalSection.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
+import { Trans } from '@lingui/macro'
```
Comment (aeolianeth) [inline src/components/shared/Project/ProjectToolsDrawer/V1TokenMigrationSection/V1TokenMigrationModal/AddTerminalSection.tsx:1]: This component renders this UI:

Screen Shot 2022-07-02 at 7 17 23 PM

File: src/components/shared/Project/ProjectToolsDrawer/ProjectToolsDrawer.tsx [hunk 3]
```diff
@@ -71,39 +73,41 @@ export function ProjectToolsDrawer({
...
-              </Trans>
-            </p>
-          </section>
-        )}
-      </Space>
-    )
-  }
+      <Divider />
+
+      <V1TokenMigrationSection />
```
Comment (aeolianeth) [inline src/components/shared/Project/ProjectToolsDrawer/ProjectToolsDrawer.tsx:108]: Add new section to the Owner tab in the Tools drawer.

File: src/components/shared/Project/ProjectToolsDrawer/V1TokenMigrationSection/V1TokenMigrationModal/StepSection.tsx [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import { Tooltip } from 'antd'
+import { ThemeContext } from 'contexts/themeContext'
+import { PropsWithChildren, useContext } from 'react'
+import { CheckCircleFilled } from '@ant-design/icons'
+import { Trans } from '@lingui/macro'
+
+export function StepSection({
```
Comment (aeolianeth) [inline src/components/shared/Project/ProjectToolsDrawer/V1TokenMigrationSection/V1TokenMigrationModal/StepSection.tsx:7]: Compone used in each section of the set up modal.

File: src/components/v2/V2Project/V2FundingCycleSection/ReservedTokensSplitsCard.tsx [hunk 5]
```diff
@@ -169,6 +168,7 @@ export default function ReservedTokensSplitsCard({
                 size="small"
                 onClick={() => setEditTokenAllocationModalVisible(true)}
                 icon={<SettingOutlined />}
+                style={{ marginBottom: '1rem' }}
```
Comment (aeolianeth) [inline src/components/v2/V2Project/V2FundingCycleSection/ReservedTokensSplitsCard.tsx:171]: Unrelated fix.

File: src/components/shared/Project/ProjectToolsDrawer/V1TokenMigrationSection/V1TokenMigrationModal/V1TokenMigrationModal.tsx [hunk 6]
```diff
@@ -0,0 +1,28 @@
+import { Trans } from '@lingui/macro'
```
Comment (aeolianeth) [inline src/components/shared/Project/ProjectToolsDrawer/V1TokenMigrationSection/V1TokenMigrationModal/V1TokenMigrationModal.tsx:1]: Full modal UI:

Screen Shot 2022-07-02 at 7 32 35 PM

File: src/components/shared/Project/ProjectToolsDrawer/V1TokenMigrationSection/V1TokenMigrationModal/StepSection.tsx (hunk 4 above)
Comment (aeolianeth) [reply src/components/shared/Project/ProjectToolsDrawer/V1TokenMigrationSection/V1TokenMigrationModal/StepSection.tsx:7]: Screen Shot 2022-07-02 at 7 33 35 PM

**Comment by johnnyd-eth** [comment]
Screen Shot 2022-07-03 at 2 40 53 pm

Disable and tooltip this when `allowSetTerminals` is disabled

**Review by johnnyd-eth — APPROVED** [review]
One comment but other than that LFGTM
